src/app/containers/Text/index.jsx

```diff
@@ -6,16 +6,11 @@ import { textModelPropTypes } from '../../models/propTypes/text';
 
 const componentsToRender = { paragraph };
 
-const shouldPrependHR = typeOfPreviousBlock => typeOfPreviousBlock === 'text';
-
 const TextContainer = ({ blocks, typeOfPreviousBlock }) => {
```

Comment:
This line is now failing, as `typeOfPreviousBlock` is not used.

```suggestion
const TextContainer = ({ blocks }) => {
```

Unfortunately I guess you'll have to change where `TextContainer` is called?